modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb

```diff
@@ -28,20 +28,25 @@ def perform(uuid, appointment_date)
 
       logger.info("Submitting travel claim for #{uuid}, #{appointment_date}")
 
-      claims_resp = TravelClaim::Service.build(
-        check_in: check_in_session,
-        params: { appointment_date: }
-      ).submit_claim
-
-      claim_number = claims_resp.dig(:data, :claimNumber)&.last(4)
+      claims_resp =
+        begin
+          TravelClaim::Service.build(
+            check_in: check_in_session,
+            params: { appointment_date: }
+          ).submit_claim
+        rescue
```

Comment:
Should the code continue after the API call fails? If no, you should add a `return unless claim_resp` after this begin/rescue block.